fc/fullcontact.go

```diff
@@ -132,8 +132,12 @@ func sendToChannel(ch chan *APIResponse, response *http.Response, url string, er
 			setTagsResponse(apiResponse)
 		case audienceCreateUrl, audienceDownloadUrl:
 			setAudienceResponse(apiResponse)
-		case emailVerificationUrl:
-			setEmailVerificationResponse(apiResponse)
+		case permissionFindUrl:
+			setPermissionFindResponse(apiResponse)
+		case permissionCurrentUrl:
+			setPermissionCurrentResponse(apiResponse)
+		case permissionVerifyUrl:
+			setPermissionVerifyResponse(apiResponse)
```

Comment:
Where are you setting response for permission create and delete?

Reply:
Added response for create and delete